k3d/README.md

```diff
@@ -0,0 +1,72 @@
+# K3d (Kubernetes in Docker)
+
+run K8s clusters locally with an image registry `localhost:5000`
```

Comment:
K3s :)

[K3s](https://github.com/k3s-io/k3s) lightweight Kubernetes distribution

for details, you can refer to  [K3s Explained](https://traefik.io/glossary/k3s-explained/)